src/game_api/string_hashes.cpp

```diff
@@ -1963,3 +1970,8 @@ const std::map<uint32_t, STRINGID> string_hashes = {
     {0x13aced89, 1959},
     {0x796b612f, 1960},
 };
+
+const std::map<uint32_t, STRINGID>& get_string_hashes()
+{
+    return string_hashes;
+}
```

Comment:
This is an auto generated file so would be better to make this change in the generator (also no empty line at the end)

Reply:
Wasn't aware that this was generated. Which file generates it?
I will add a comment to make sure that is more obvious in the future.

Reply:
`strings_get_hashes.py`, we can also make it output to console, so you would just copy paste the array instead of generating the whole file